STAR-Delegation/CSR-template/template-schema.cddl

```diff
@@ -40,25 +40,25 @@ $signatureType /= "sha256WithRSAEncryption"
 
 ecKeyType = {
   PublicKeyType: "ecPublicKey"
-  ? Curve: $ecCurve
-  ? SignatureType: $ecSignatureType
+  Curve: $ecCurve
+  SignatureType: $ecSignatureType
```

Comment:
I thought you wanted to add Mr. Edwards.

Reply:
thanks for the nudge!